search_test.go

```diff
@@ -1889,3 +1889,47 @@ func TestHightlightingWithHTMLCharacterFilter(t *testing.T) {
 			gotFragment, expectedFragment)
 	}
 }
+
+func TestIPRangeQuery(t *testing.T) {
+	idxMapping := NewIndexMapping()
+	im := NewIPFieldMapping()
+	dmap := mapping.NewDocumentMapping()
+	dmap.AddFieldMappingsAt("ip_content", im)
+	idxMapping.DefaultMapping = dmap
+
+	tmpIndexPath := createTmpIndexPath(t)
+	defer cleanupTmpIndexPath(t, tmpIndexPath)
+
+	idx, err := New(tmpIndexPath, idxMapping)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	defer func() {
+		err = idx.Close()
+		if err != nil {
+			t.Fatal(err)
+		}
+	}()
+
+	ipContent := "1.2.3.4"
```

Comment:
A small doubt, do you think we can use a dummy (but a more realistic) IP address over here as an example?

Reply:
Ack.